### PR TITLE
Add user_active checkbox

### PR DESCRIPTION
**Pull request by jesusantguerrero**
- Add user status chceckbox

### CODE REVIEW

File: src/components/companies/users/form.vue [hunk 1]
```diff
@@ -119,7 +136,7 @@
                     Cancel
                 </button>
                 <button
-                    :disabled="isLoading || !hasChanged"
```
Comment (SparoHawk) [inline src/components/companies/users/form.vue:122]: Why did you remove the `!hasChanged` variable?

File: src/components/companies/users/form.vue [hunk 2]
```diff
@@ -110,6 +110,23 @@
                                 />
                                 <span class="text-danger">{{ errors.first("role") }}</span>
                             </div>
+
+                            <div class="form-group">
+                                <label> Status </label>
```
Comment (SparoHawk) [inline src/components/companies/users/form.vue]:
```suggestion
                                <label>Status</label>
```

File: src/components/companies/users/form.vue [hunk 3]
```diff
@@ -110,6 +110,23 @@
...
+                                <div>
+                                    <label for="user_active">
```
Comment (SparoHawk) [inline src/components/companies/users/form.vue]:
```suggestion
                                    <label>
```

`for` is not necessary if the `input` is inside the `label`.

File: src/components/companies/users/form.vue [hunk 4]
```diff
@@ -110,6 +110,23 @@
...
+                                        <input 
+                                          type="checkbox" 
+                                          name="user_active" 
+                                          v-model="userData.user_active" 
+                                          id="user_active"
```
Comment (SparoHawk) [inline src/components/companies/users/form.vue]:
```suggestion
```

File: src/components/companies/users/form.vue [hunk 5]
```diff
@@ -215,7 +233,7 @@ export default {
         },
         setRole(value) {
             this.userData.roles_id = value.id;
-        },
+        },
```
Comment (SparoHawk) [inline src/components/companies/users/form.vue]:
```suggestion
        },
```

File: src/components/companies/users/form.vue [hunk 6]
```diff
@@ -207,6 +224,7 @@ export default {
             await axios({
                 url: `/users/${this.$route.params.id}`
             }).then(({ data }) => {
+                data.user_active = !!+data.user_active;
```
Comment (SparoHawk) [inline src/components/companies/users/form.vue]:
```suggestion
                data.user_active = Boolean(Number(data.user_active));
```

File: src/components/companies/users/form.vue [hunk 7]
```diff
@@ -231,6 +249,7 @@ export default {
                 data.append("role_id", this.selectedRole.id);
             } else {
                 data = this.userData;
+                data.user_active = +data.user_active;
```
Comment (SparoHawk) [inline src/components/companies/users/form.vue]:
```suggestion
                data.user_active = Number(data.user_active);
```

**Review by SparoHawk — CHANGES_REQUESTED** [review]

File: src/components/companies/users/form.vue (hunk 1 above)
Comment (jesusantguerrero) [reply src/components/companies/users/form.vue:122]: we have no validation for the new field `user_active` because is not required it can be `false|true` so hasChanged not change if I just change this user_active.

**Review by SparoHawk — APPROVED** [review]
